users/service.go

```diff
@@ -159,21 +158,23 @@ type UserPage struct {
 var _ Service = (*usersService)(nil)
 
 type usersService struct {
-	users  UserRepository
-	groups GroupRepository
-	hasher Hasher
-	email  Emailer
-	auth   mainflux.AuthServiceClient
+	users        UserRepository
+	groups       GroupRepository
+	hasher       Hasher
+	email        Emailer
+	auth         mainflux.AuthServiceClient
+	uuidProvider mainflux.IDProvider
```

Comment:
This is not necessarily a UUID provider (we'll switch to ULID soon) and the underlying format should be transparent to the Users service (that's one of the reasons we use the interface), so `idp` or `idProvider` sounds like a better naming.

Reply:
Right, let's follow interface naming.